ding/bonus/dqn.py

```diff
@@ -259,6 +374,21 @@ def batch_evaluate(
             context: Optional[str] = None,
             debug: bool = False
     ) -> EvalReturn:
+        """
+        Overview:
+            Evaluate the agent with DQN algorithm for ``n_evaluator_episode`` episodes with ``env_num`` evaluator \
```

Comment:
also describe the different usage of `batch_evaluate` and `deploy`

Reply:
fixed.